molecules/_dropdown.scss

```diff
@@ -127,18 +139,33 @@ $dropdown-background-color-active: $arctica !default;
       }
     }
     .dropdown-item {
-      list-style: none;
       .text {
         @include dropdown-item(
                 $dropdown-font-color,
                 $dropdown-font-family,
-                $dropdown-background-color-accent,
-                $dropdown-background-color-active
+                $dropdown-color-active,
+                $dropdown-background-color-accent
         );
         .icon {
-          margin-right: rem(5px);
+          width: $space-m;
+          margin-right: $space-s;
         }
       }
+      &.has-children > .text:after {
+        content: map-get($icons, right);
+        @include icon-base();
+        font-size: rem(16px);
```

Comment:
Actually I don't really like the solution in general since it just put's an icon into a :after this should be easier like with an mixin. I gonna fix it